### ISSUE TITLE
[BUG] NameError when calling import_pbp_data function due to 'HTTPError in exception handling.

### ISSUE THREAD

**Issue by Laneville** [nflverse/nfl_data_py#99]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you installed the latest development version of the package(s) in question?

- [X] I have installed the latest development version of the package.

### If this is a data issue, have you tried clearing your nflverse cache?

I have cleared my nflverse cache and the issue persists.

### What version of the package do you have?

0.3.3

### Describe the bug

When trying to run the pbp data fetch function (nfl.import_pbp_data(years, columns, downcast=True, cache=False, alt_path=None))
I'm receiving an HTTPError and a NameError is raised as a result

### Reprex

```r
import nfl_data_py as nfl
pbp_data = nfl.import_pbp_data(years=[2024])
```

### Expected Behavior

nfl.import_pbp_data should return a dataframe of the data for the year specified

### nflverse_sitrep

```r
NA
```

### Additional context

Doing some tracing, I believe this may be a data issue with where it's fetching the files from 'nflverse-data'. Could be that this dataset maybe isn't updating correctly and the URL isn't found as result

**Comment by cdeez-oss** [comment]
I am also running into the same problem with 2024 data. However I am on version 0.3.1, I had figured it may be a problem with the older package.

**Comment by alecglen** [comment]
Hi there, thanks for the report!

`import_pbp_data()` is erroring currently for 2024 due to the [loss of the pbp_participation dataset](https://x.com/John_B_Edwards/status/1832091502895579354). I'll have a fix in the 0.3.3 release soon, but in the mean time you can avoid the error by calling the function with the optional parameter `include_participation=False`.

**Comment by Laneville** [comment]
Thanks @alecglen ! this worked perfectly.

**Comment by cdeez-oss** [comment]
Thank you @alecglen! this also works for the older package for anyone else running into the error.

**Comment by roe-men** [comment]
also running into this error and tried the above but haven't been able to fix..

**Comment by alecglen** [comment]
@roe-men can you paste the full stack trace?

**Comment by roe-men** [comment]
An error occurred while pulling the play-by-play data: name 'Error' is not defined
Traceback (most recent call last):
  File "/Users/robertomendoza/nfl_folder/venv/lib/python3.9/site-packages/nfl_data_py/__init__.py", line 147, in import_pbp_data
    partic = pandas.read_parquet(path)
  File "/Users/robertomendoza/nfl_folder/venv/lib/python3.9/site-packages/pandas/io/parquet.py", line 667, in read_parquet
    return impl.read(
  File "/Users/robertomendoza/nfl_folder/venv/lib/python3.9/site-packages/pandas/io/parquet.py", line 396, in read
    handles = get_handle(
  File "/Users/robertomendoza/nfl_folder/venv/lib/python3.9/site-packages/pandas/io/common.py", line 728, in get_handle
    ioargs = _get_filepath_or_buffer(
  File "/Users/robertomendoza/nfl_folder/venv/lib/python3.9/site-packages/pandas/io/common.py", line 384, in _get_filepath_or_buffer
    with urlopen(req_info) as req:
  File "/Users/robertomendoza/nfl_folder/venv/lib/python3.9/site-packages/pandas/io/common.py", line 289, in urlopen
    return urllib.request.urlopen(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 523, in open
    response = meth(req, response)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 632, in http_response
    response = self.parent.error(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 561, in error
    return self._call_chain(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/robertomendoza/nfl_folder/play_by_play.py", line 214, in main
    pbp_data = nfl.import_pbp_data(years=years, columns=columns, downcast=True)
  File "/Users/robertomendoza/nfl_folder/venv/lib/python3.9/site-packages/nfl_data_py/__init__.py", line 153, in import_pbp_data
    except Error as e:
NameError: name 'Error' is not defined

**Comment by alecglen** [comment]
> pbp_data = nfl.import_pbp_data(years=years, columns=columns, downcast=True)

That 4th line from the bottom is from your code, and I see you're not using the `include_participation=False` argument as described above. Did you perhaps forget to save the file change?

**Comment by roe-men** [comment]
that worked. thanks!

**Comment by alecglen** [comment]
Fixed in 0.3.3. This will be available via PyPI soon, but can be installed now with the command below if needed.
```
pip install --force-reinstall nfl-data-py@git+https://github.com/nflverse/nfl_data_py.git@v0.3.3
```
